tests/rptest/wasm/topics_result_set.py

```diff
@@ -31,10 +31,10 @@ class TopicsResultSet:
     A high level wrapper around a result set from a read of a set of kafka
     topic partitions.
     """
-    def __init__(self, rset={}):
+    def __init__(self, rset=None):
```

Comment:
per the commit subject the parameter is not used. should it be removed?

Reply:
ah nice catch, for sure